src/main/java/Ryan.java

```diff
@@ -24,9 +24,9 @@ public static void main(String[] args) {
                     System.out.println((i + 1) + "." + tasks.get(i));
                 }
                 Utils.horizontalLine();
-            } else if (command.startsWith("mark")){
+            } else if (command.startsWith("mark")) {
                 int index = Integer.parseInt(command.split(" ")[1]) - 1;
-                if (index >= 0 && index < tasks.size()){
+                if (index >= 0 && index < tasks.size()) {
```

Comment:
Can create a new boolean variable E.g. isOutOfBounds, to store the condition to improve code readability